index.html

```diff
@@ -425,11 +425,12 @@ <h2>LDP-NRs</h2>
         <h2>HTTP PUT</h2>
         <p>
           When accepting a <code>PUT</code> request against an existing resource, an HTTP <code>Link: rel="type"</code>
-          header MAY be included. If the type in the Link header is a subtype of a current type of the resource, and
-          has an interaction model assigned to it by [[!LDP]], then the resource MUST be assigned the new type and the
-          interaction model of the resource MUST be changed to the interaction model assigned to the new type by
-          [[!LDP]]. If that type is a value in the LDP namespace and is not either a current type of the resource or a
-          subtype of a current type of the resource, the request MUST be rejected with a 409 (Conflict) response.
+          header MAY be included. If the type in the <code>Link</code> header is a subtype of a current type of the
```

Comment:
Is there any way we can break this monster sentence up?  Seems like there's a lot going on, and we need to suss out the difference between assigning type and assigning interaction models.

Reply:
@barmintor @escowles @awoods @zimeon ^^

Reply:
"type" is a little ambiguous here when what we're talking about is interaction models, or "LDP types". "LDP types" may even be preferable, since it avoids blanketing anything that comes back in `Link@rel=type`.

Reply:
"LDP types MAY be changed to a new LDP type that is rdfs:subClassOf the current type."

Reply:
Also a little wary of specifying the mechanism here, since it's not in LDP at all.

Reply:
I can't wait to hear what @escowles and @zimeon say.

Reply:
Here is the only comment I see in the LDP spec about changes of interaction model:

> Non-normative note: LDP assumes that the interaction model of a resource is fixed when the resource is created, and this is reflected in the language of the bullets. If an implementation were to extend LDP by allowing the interaction model to vary after creation, that is viewed as a compatible extension to LDP and the statements above would constrain the default interaction model rather than saying that no other behavior is possible.

I agree with @barmintor that the current statement is too vague, something like his suggestion "LDP types MAY be changed to a new LDP type that is rdfs:subClassOf the current type." would seem appropriate -- but this raises the issue of MAY vs MUST. Do we have an understood use case that would make us require implementations to support changes of interaction model?